src/Debug/debugLayer.ts

```diff
@@ -121,6 +125,17 @@ export class DebugLayer {
      */
     public static InspectorURL = `https://unpkg.com/babylonjs-inspector@${Engine.Version}/babylon.inspector.bundle.js`;
 
+    /**
+     * Action tab kinds
+     */
+    public static ACTION_TAB = {

```

Comment:
why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class

Reply:
do you mean that export enum from debugLayer.ts, then inspector.ts import that enum?

Reply:
> 
> 
> why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class

imo, the debuglayer is a consumer of inspector.. what tabs are available in debuglayer should be the tabs exporting from inspector.. should not be opposite, what do you think?

Reply:
> 
> 
> > why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class
> 
> imo, the debuglayer is a consumer of inspector.. what tabs are available in debuglayer should be the tabs exporting from inspector.. should not be opposite, what do you think?

the reason why i can't do that has mentioned at https://github.com/BabylonJS/Babylon.js/pull/7235#discussion_r352292106 :D

Reply:
Yes this is weird I agree but we are already doing it. It is ok to expose it and import it from the inspector

Reply:
ok.